src/validators.ts

```diff
@@ -2,6 +2,7 @@ import spawn from "cross-spawn";
 import { ChildProcess } from "node:child_process";
 import { join, resolve } from "node:path";
 import { readFileSync, writeFileSync } from "node:fs";
+import { getBinPath } from "@tableland/validator";
```

Comment:
Aside from building and holding the actual binary, this package's main function is exposing a method that will return the path to the correct binary for the running process.